src/chapters/radius-auth.xml

```diff
@@ -0,0 +1,251 @@
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 1:20.134s
+[INFO] Finished at: Wed Jan 31 09:45:41 EST 2018
+[INFO] Final Memory: 47M/749M
+[INFO] ------------------------------------------------------------------------</computeroutput>
+<prompt>$</prompt></screen>
+        </informalexample>
+
+        <para>After the build completes successfully, the extension will be in the
+            <filename>extensions/guacamole-auth-radius/target/</filename> directory, and will be
+            called guacamole-auth-radius-0.9.14.jar.  This extension file can be copied to
+            the <filename>GUACAMOLE_HOME/extensions</filename> directory.
+            <emphasis>If you are unsure where <varname>GUACAMOLE_HOME</varname> is located on
+            your system, please consult <xref linkend="configuring-guacamole"/> before
+            proceeding.</emphasis></para>
+
+        <para>Extensions are loaded in alphabetical order, and authentication is performed
+            in the order in which the extensions were loaded.  If you are stacking the
+            RADIUS extension with another extension, like the JDBC extension, in order to
+            store connection information, you may need to change the name of the RADIUS
+            extension such that it is evaluated prior to the JDBC extension - otherwise
+            an authentication failure in one of the previous modules may block the RADIUS
+            module from ever being evaluated.</para>
+
+        <para>To install the RADIUS authentication extension, you must:</para>
+        <procedure>
+            <step>
+                <para>Create the <filename>GUACAMOLE_HOME/extensions</filename> directory, if it
+                    does not already exist.</para>
+            </step>
+            <step>
+                <para>Copy <filename>guacamole-auth-radius-0.9.14.jar</filename> into
+                        <filename>GUACAMOLE_HOME/extensions</filename>.</para>
+            </step>
+            <step>
+                <para>Configure Guacamole to use RADIUS authentication, as described
+                    below.</para>
+            </step>
+        </procedure>
+    </section>
+    <section xml:id="guac-radius-config">
+        <title>Configuring Guacamole for RADIUS authentication</title>
+        <indexterm>
+            <primary>configuring RADIUS authentication</primary>
+        </indexterm>
+        <indexterm>
+            <primary>RADIUS authentication</primary>
+            <secondary>configuration</secondary>
+        </indexterm>
+        <para>This extension provides several configuration properties in order 
+            to communicate properly with the RADIUS server to which it needs to authenticate.  It is 
+            important that you know several key pieces of information about the RADIUS server - 
+            at a minimum, the server name or IP, the Authentication port, the authentication 
```

Comment:
Why is "authentication" capitalized here?

Reply:
Fixed.